src/index.html

```diff
@@ -1,13 +1,58 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__photo">
+        <img
+          class="product-card__img"
+          src="../src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+      </div>
+
+      <article class="product-card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </article>
+
+      <div class="product-card__code">
+        <p>Product code: 195434</p>
+      </div>
+
+      <div class="product-card__stars">
+        <div class="product-card__star"></div>
+        <div class="product-card__star"></div>
+        <div class="product-card__star"></div>
+        <div class="product-card__star"></div>
+        <div class="product-card__star"></div>
+
+        <p class="product-card__review">Reviews: 5</p>
+      </div>
+
+      <div class="product-card__price-title">
+        <p>Price:</p>
+        <p class="product-card__price">$2,199</p>
+      </div>
+
+      <a
+        class="product-card__buy-link"
+        href="buy"
+        data-qa="hover"
+        >
+        Buy
+      </a>
```

Comment:
```suggestion
      <a
        class="product-card__buy-link"
        href="buy"
        data-qa="hover"
      >
        Buy
      </a>
```
checklist:
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/a11c55ea-6246-465e-bbcf-dfa3d6294056)